modules/installation-rhv-an-example-inventory-yml-file.adoc

```diff
@@ -0,0 +1,237 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-an-example-inventory-yml-file_{context}"]
+= Example inventory.yml file
+
+This example `inventory.yml` shows the variables needed for the UPI installation process, followed by a explanations of each one.
+
+.Example `inventory.yml` file
+[source,yaml]
+----
+---
+all:
+  vars:
+
+    ovirt_cluster: "Default" <1>
+    ocp:
+      assets_dir: "{{ lookup('env', 'ASSETS_DIR') }}" <2>
+      ovirt_config_path: "{{ lookup('env', 'HOME') }}/.ovirt/ovirt-config.yaml" <3>
+
+    # ---
+    # RHCOS section
+    # ---
+    rhcos:
+      image_url: "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.6/rhcos-4.6.0-0.nightly-2020-07-16-122837-x86_64-openstack.x86_64.qcow2.gz" <4>
+      local_cmp_image_path: "/tmp/rhcos.qcow2.gz" <5>
+      local_image_path: "/tmp/rhcos.qcow2" <6>
+
+    # ---
+    # Profiles section
+    # ---
+    control_plane:
+      cluster: "{{ ovirt_cluster }}"
+      memory: 16GiB
+      sockets: 4
+      cores: 1
+      template: rhcos_tpl
+      operating_system: "rhcos_x64"
+      type: high_performance
```

Comment:
I would change that to from "high perfomance" to "server", first because a known bug - which I know it's gonna be fixed soon, but, and second, the user that will set "high perfomance" will have to set others variables that are not specified here. It's an specific case that whoever needs will know how to do it.

Reply:
Or maybe just add a bubble number note (like  <7>) saying that manual changes need to be made for high perfomance type and referce this: https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/virtual_machine_management_guide/configuring_high_performance_virtual_machines_templates_and_pools

Reply:
I know it's an example, but an unaware user could be mislead here IMHO